.travis.yml

```diff
@@ -1,7 +1,11 @@
 language: php
+
 php:
-  - "5.4"
-  - "5.5"
+  - 5.4
+  - 5.5
+  - 5.6
+  - 7.0
+  - hhvm
```

Comment:
I don't think it's necessary to support HHVM. However we should run tests with PHP 7.1 and 7.2.

Reply:
I did not touch those files.